src/main/webapp/app/shared/badge/count-badge.scss

```diff
@@ -0,0 +1,25 @@
+.oncokb.icon {
+  margin: 0;
+}
+
+.count-badge-wrapper {
+  font-size: 90%;
```

Comment:
Is there a way to shrink the size of the oncokb icons from oncokb-styles? I can make the text's font size smaller, if I can adjust the icon's size, but I don't see a way to do that.

Another option is that I can make the mutation name and all other bolded text `h4`, but keep the default font-size for all other text.

Reply:
@calvinlu3 there is no easy way I imagine because they from an image. Ideally, we should create our own font lib to use this icons. We can leave them as it for now.